backend/src/controllers/websockets_controller.ts

```diff
@@ -1,9 +1,13 @@
 import WebSocket from "ws";
 import { RequestWithCache } from "../middlewares/app_cache";
 
-enum MessageType {
+export enum InboundMessageType {
   PING = "PING",
   JOIN_GAME = "JOIN_GAME",
+}
+
+export enum OutboundMessageType {
+  WAITING_ROOM_UPDATE = "WAITING_ROOM_UPDATE",
   SUCCESS = "SUCCESS",
```

Comment:
Should this be removed ? It doesn't seemed to be used anymore.
```suggestion
```

Reply:
Indeed :) I commit the suggestion